components/basecomponents/StoryReader.tsx

```diff
@@ -56,6 +56,13 @@ const StoryReader = ({ titleCurrent, titleOther, id, phrases }: StoryReaderProps
     });
     setLanguagePlay(language);
   };
+  const plausible = usePlausible();
+  const filePathStory = `/pdf/${id}-${currentLanguage}.pdf`;
+
+  const handleDownloadStory = () => {
```

Comment:
Looks great!🤩 I propose having a function `handleDownload` with parameters `language`, `url`, `category`, but we can add these cosmetic changes later.